doc/doxygen/headers/constraints.h

```diff
@@ -432,4 +432,59 @@
  * Either behavior can also be achieved by building two separate
  * ConstraintMatrix objects and calling ConstraintMatrix::merge function with
  * a particular second argument.
+ *
+ *
+ * <h3>Applying constraints indirectly with a LinearOperator</h3>
+ *
+ * Sometimes it is either not desirable, or not possible to directly
+ * condensate, or eliminate constraints from a system of linear equations.
+ * In particular if there is no underlying matrix object that could be
+ * condensed (or taken care of constraints during assembly). This is
+ * usually the case if the system is described by a LinearOperator.
+ *
+ * In this case we can solve the modified system
+ * @f[
+ *   (C^T A C + Id_c) \tilde x = C^T (b - A\,k)
+ * @f]
+ * instead [1]. Here, $A$ is a given (unconstrained) system matrix $A$ and
+ * $b$ the corresponding right hand side of a system of linear equations
+ * $A\,x=b$. The matrix $C$ describes the homogeneous part of the linear
+ * constraints stored in a ConstraintMatrix and the vector $k$ is the
+ * vector of corresponding inhomogeneities. More precisely, the
+ * ConstraintMatrix::distribute() operation applied on a vector $x$ is the
+ * operation
+ * @f[
+    x \leftarrow C\,x+k.
```

Comment:
What I mean is that in https://www.dealii.org/developer/doxygen/deal.II/classConstraintMatrix.html we describe the matrix that is associated with a `ConstraintMatrix` object as having size `m` by `n` where `m` equals the number of constraints. So multiplying with this matrix would yield a vector of size `m`, but the way you explain the matrix `C` here is seems to me that it's `n` by `n` where the rows corresponding to unconstrained degrees of freedom have only the diagonal entry set to one. 

I'm just pointing out that we describe "the matrix associated with the ConstraintMatrix object" in different ways here and in the class documentation.